tests/metrics/test_pcr_metrics.py

```diff
@@ -42,5 +79,5 @@ def test_pcr_batch_embedding(adata):
         n_comps=50,
         scale=True,
     )
-    LOGGER.info(f"using embedding: {score}")
+    LOGGER.info(f"using X_emb: {score}")
```

Comment:
Why do you use proper logging in tests but simple prints in the code? Would be nice to use loggers everywhere :)

Reply:
Good question, the testing code was obviously written way later than the actual function code. This would require refactoring all functions. I'd move that to when I have more time (since I feel like that would be way lower prio)